src/install.rs

```diff
@@ -3,6 +3,8 @@ use crate::install_frame::{InstanceSource, InstanceSourceType};
 use crate::instance::{Instance, InstanceType};
 use crate::{archive, github};
 use anyhow::Result;
+use dmg::Attach;
```

Comment:
`Attach` is rather ambiguous, you might want to `use dmg` and then later use `dmg::Attach`.

Reply:
Completely agree, was not aware it can be done that way